initial-ai-src/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ruter Reis Discount Calculator</title>
+    <link rel="stylesheet" href="style.css" />
```

Comment:
:art: no big deal, but I prefer to use paths with clear file separators (i.e `/style.css` or `./style.css` rather than `style.css`). There's less ambiguity (especially for those unfamiliar with html) to be a bit clearer. Most developers know unix paths.